text/3-entrees/storage-api/enumerated.md

```diff
@@ -104,7 +104,7 @@ fn remove_member_bounded(origin, index: u32) -> Result {
 
 ### Linked Map <a name = "linkedmap"></a>
 
-To trade performance for *relatively* simple code, use the `linked_map` data structure. By implementing [`StorageLinkedMap`](https://substrate.dev/rustdocs/master/frame_support/storage/trait.StorageLinkedMap.html) in addition to [`StorageMap`](https://substrate.dev/rustdocs/master/frame_support/storage/trait.StorageMap.html), `linked_map` provides a method `head` which yields the head of the *list*, thereby making it unnecessary to also store the `LargestIndex` (the *counters*). The `enumerate` method also returns an `Iterator` ordered according to when `(key, value)` pairs were inserted into the map.
+To trade performance for *relatively* simple code, use the `linked_map` data structure. By implementing `StorageLinkedMap` in addition to [`StorageMap`](https://substrate.dev/rustdocs/master/frame_support/storage/trait.StorageMap.html), `linked_map` provides a method `head` which yields the head of the *list*, thereby making it unnecessary to also store the `LargestIndex` (the *counters*). The `enumerate` method also returns an `Iterator` ordered according to when `(key, value)` pairs were inserted into the map.
```

Comment:
The changes in this file appear to have nothing to do with the rest of this PR; why are they here?

Reply:
Yeah, it's totally separate. They're here to make the CI green on this PR and as a stopgap until this page is removed entirely. It was some bad git usage :-/

From that commit's message:

> This is a stopgap measure to remove broken links. This article has been 
properly removed in the follow-substrate-master branch, but not merged 
here yet :-/